OrbitGl/ProcessManager.h

```diff
@@ -42,13 +42,17 @@ class ProcessManager {
   // Get a copy of process list.
   virtual std::vector<ProcessInfo> GetProcessList() const = 0;
 
+  virtual outcome::result<std::string, grpc::Status, outcome::policy::terminate>
```

Comment:
Apparently I need to specify the `NoValuePolicy` explicitly when using `grpc::Status` as error type...

Reply:
That's expected, since it cannot be thrown as an exception or converted into an error code automatically.

Reply:
This is leaking grpc-specific class outside of manager. I am not sure if we want this... I guess the alternative is to replace this with Error.

Reply:
So should I just go with something like `struct Error { std::string value; }`?

Reply:
I think it is better.

Reply:
Done.